.coderabbit.yml

```
@@ -0,0 +1,46 @@
+
+pr_template:
+  required: true
+  sections:
+    - description: "Proposed changes"
+    - types_of_changes: "Types of changes"
+    - checklist: "Checklist"
+    - dependant_prs: "Dependant PRs"
+    - relevant_issues: "Relevant Issues"
+    - detailed_description: "Description"
+
+ignore_draft_pr: true
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)